CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+We mostly adhere to [Keep a Changelog 1.1.0](https://keepachangelog.com/en/1.0.0/)
+but use the following change types: "Added", "Changed", "Fixed" & "Dependencies Updated".
```

Comment:
We could also use backticks for the types, like:

````
``Added``, ``Changed``, ``Fixed`` & ``Dependencies Updated``.
````

Reply:
@ravage84I think using regular quotation marks is sufficient

Reply:
@TerryKern I agree.